.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - feature
+  - request
+# Label to use when marking an issue as stale
+staleLabel: wontfix
```

Comment:
Maybe use `outdated` label? I created `wontfix` label to mark issue that won't be resolved due to some reasons.